app/javascript/components/widgets/weather/semi-circle.jsx

```diff
@@ -207,6 +221,6 @@ const SunCircle = styled.circle`
 offset-path: path('${p => p.arcPath}');
 animation: ${p => p.arcAnimation} 1000ms 1 normal forwards ease-in-out;
 r: ${p => p.ballRadius};
-fill: ${colors.pink}`;
+fill: ${p => p.fillStyle}`;
```

Comment:
I think for svgs we can `fill: currentColor` and we will keep the color from the parent element.

https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword